python/core/auto_generated/qgstaskmanager.sip.in

```diff
@@ -169,7 +169,7 @@ be canceled if any of these layers are about to be removed.
 .. seealso:: :py:func:`setDependentLayers`
 %End
 
-    bool waitForFinished( unsigned long timeout = 30000 );
+    bool waitForFinished( int timeout = 30000 );
```

Comment:
What's the advantage of int ulong -> int?

Reply:
tryLock takes an int, so it would be misleading to leave this as the ulong. (Qt API is at fault here!)